docs/getting-started/endpoint-management-req.asciidoc

```diff
@@ -1,4 +1,29 @@
 [[endpoint-management-req]]
 = Endpoint management requirements
 
-This page is a placeholder for future documentation.
+You can create user roles and define privileges to manage feature access in {kib}. This allows you to manage access to {elastic-defend}'s endpoint management features using the principle of least privilege.
+
+Roles and privileges are configured in *Stack Management* -> *Roles* in {kib}. For more details on using this UI, refer to {kibana-ref}/kibana-role-management.html#adding_kibana_privileges[{kib} privileges]. 
+
+.Requirement
+[sidebar]
+--
+Endpoint management feature privileges must be assigned to *All Spaces*. You can't assign them to individual {kib} spaces. 
+--
+
+To grant access, set the {kib} privileges for the following sub-features under the *Security* feature in the *{kib} privileges* configuration UI:
```

Comment:
I actually started with a screenshot, but removed it for future maintenance because I know this UI will change a lot in future releases. But I agree, it's better for the user to have an illustration here, so I'll add it back.